awx/main/analytics/collectors.py

```diff
@@ -217,30 +218,34 @@ def job_instance_counts(since):
     return counts
 
 
-@register('query_info', '1.0')
-def query_info(since, collection_type):
+@register('query_info', '1.0', description=_('Metadata about the analytics collected'))
+def query_info(since, collection_type, **kwargs):
     query_info = {}
     query_info['last_run'] = str(since)
     query_info['current_time'] = str(now())
```

Comment:
this should probably have the gather time? Although we don't use it anywhere

Reply:
Updated.